### PR TITLE
fix e2e error reclaim

### LINKED ISSUE #360: Reclaim e2e test failed
The e2e test of recliam keep failing, refer to https://travis-ci.org/kubernetes-incubator/kube-arbitrator/builds/426008715 for detail.

### PR DESCRIPTION
**Pull request by denkensk**
fix #360
If we set NamespaceAsQueue to true, kube-batchd will not handle Queue CRD anymore.  Then Reclaim in e2e test will be failed to use queue "q1" and "q2".

### CODE REVIEW

**Comment by denkensk** [comment]
cla change

File: cmd/kube-batchd/app/options/options.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func (s *ServerOption) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.SchedulerConf, "scheduler-conf", "", "The namespace and name of ConfigMap for scheduler configuration")
 	fs.BoolVar(&s.EnableLeaderElection, "leader-elect", s.EnableLeaderElection, "Start a leader election client and gain leadership before "+
 		"executing the main loop. Enable this when running replicated kar-scheduler for high availability")
-	fs.BoolVar(&s.NamespaceAsQueue, "enable-namespace-as-queue", true, "Make Namespace as Queue with weight one, "+
+	fs.BoolVar(&s.NamespaceAsQueue, "enable-namespace-as-queue", s.NamespaceAsQueue, "Make Namespace as Queue with weight one, "+
```
[the remark this answers is not in the document]
Comment (k82cn) [reply cmd/kube-batchd/app/options/options.go:51]: For example, add a parameter here: https://github.com/kubernetes-incubator/kube-arbitrator/blob/master/hack/run-e2e.sh#L14

[the remark this answers is not in the document]
Comment (denkensk) [reply cmd/kube-batchd/app/options/options.go:51]: OK I can try to do it. 
Can I create q1/qj-1 in namespace q1 and q2/qj-2 in namespace q2 in Reclaim in e2e script?
```go
It("Reclaim", func() {
		context := initTestContext()
		defer cleanupTestContext(context)

		jobName1 := "q1/qj-1"
		jobName2 := "q2/qj-2"

		slot := oneCPU
		rep := clusterSize(context, slot)

		createJob(context, jobName1, 1, rep, "nginx", slot, nil)
```

[the remark this answers is not in the document]
Comment (k82cn) [reply cmd/kube-batchd/app/options/options.go:51]: Yes, that's better :) `NamespaceAsQueue` should be enabled by default. If the user who want advanced reclaim , e.g. weight , they can enable it.

File: test/util.go [hunk 2]
```diff
@@ -63,23 +63,23 @@ type context struct {
...
 	}
 
 	return nss[0], nss[1]
 }
 
 func initTestContext() *context {
 	cxt := &context{
-		namespace: "test",
-		queues:    []string{"q1", "q2", "test"},
+		namespaces: []string{"test", "n1", "n2"},
```
Comment (k82cn) [inline test/util.go]: Oh, how about have a parameters to create q1, q2 as namespace or Queue? We can have a random value for `enable-namespace-as-queue` to improve the coverage.

Comment (k82cn) [reply test/util.go]: env is ok to me :)

**Pull request merged**